interoperability-layer-openhim/package-metadata.json

```
@@ -19,6 +19,7 @@
     "OPENHIM_CONSOLE_CPU_RESERVE": "0.05",
     "OPENHIM_CONSOLE_MEMORY_LIMIT": "2G",
     "OPENHIM_CONSOLE_MEMORY_RESERVE": "500M",
+    "OPENHIM_CONSOLE_SHOW_LOGIN": "true",
```

```suggestion
    "OPENHIM_CONSOLE_SHOW_LOGIN": "false",
```
Is keycloak enabled by default ?

---

I think its fine to leave as is. These configs will be overridden by whats in the env files for the recipes